R/readKane2004.R

```diff
@@ -0,0 +1,69 @@
+#' Raw Read for data corresponding to the Kane dataset
+#'
+#' Download and read in the raw data tables from Kane, E., Ping, C., Chapin, S. and 
+#' Valentine, D. with Bonanza LTER. doi:10.6073/pasta/d09433eee2cb6587eca672864cf7e90f
+#' 
+#' This is not post-processed or QA/QCed by this package.
+#'
+#' @param dataDir filename for download directory
+#' @param download boolean that will download the files from repository 
+#'
+#' @return a list with meta data and raw table reads. This will be 49 KB in size.
+#' @export
+#' @importFrom readr read_csv
+
+
+  
+#data read in for citation [2] not included
+  #url confirmation is needed, see ##TODO from data_reports/114_Kane.Rmd
+#KaneBiophysicalData is missing header information that could not be found online, contacting of researchers is necessary
+#licensing information was not found, contacting of EDI is necessary
+
+
+readKane2004 <- function(dataDir, download = TRUE, verbose = FALSE){
+ 
+  
+  urlTable <- data.frame(fileBase = c('x190_2031_all_soil_profile_depths_carbon_BD.txt', 'x190_1608_sitesummarydata.txt', 'BNZeml132.xml'),
+                          downloadURL = c('http://www.lter.uaf.edu/php/download_data.php?f=/data_files/ascii/files/190_2031_all_soil_profile_depths_carbon_BD.txt', 'http://www.lter.uaf.edu/php/download_data.php?f=/data_files/ascii/files/190_1608_sitesummarydata.txt', 'http://www.lter.uaf.edu/eml/BNZeml132.xml'))
+  
+  
+  #for loop to read in data from urlTable if it does not exist in local repository
+  for(ii in 1:nrow(urlTable)){
+    dataFile <- file.path(dataDir, urlTable$fileBase[ii])
+    if(!(file.exists(dataFile))){
+      download.file(urlTable$downloadURL[ii], destfile= dataFile, quiet=FALSE)
+    }
+  }
+  
+  # #this code was not used as it requires additional package dependency
+  # plyr::d_ply(
+  #   urlTable,
+  #   c('downloadURL','fileBase'),
+  #   function(xx){
+  #     dataFile <- file.path(dataDir, xx$fileBase)
+  #     if(!(file.exists(dataFile))){
+  #     download.file(xx$downloadURL, destfile= dataFile, quiet=FALSE)
+  #     }
+  #     }
+  #   )
+  
+  #assigning filenames to a variable
+  kaneLayerdata <- file.path(dataDir, 'x190_2031_all_soil_profile_depths_carbon_BD.txt')
+  kaneSitedata <- file.path(dataDir, 'x190_1608_sitesummarydata.txt') 
+  kaneMetadata <- file.path(dataDir, 'BNZeml132.xml')
+  
```

Comment:
This file got duplicated, use `git rm` to CAREFULLY remove one or the other of these two files.

Same problem as above with the repeated name string.